.travis.yml

```diff
@@ -0,0 +1,32 @@
+dist: xenial
+arch: arm64
+os: linux
+language: go
+go:  1.13.x
+addons:
+  apt:
+    sources:
+      - sourceline: 'ppa:ubuntu-toolchain-r'
+      - llvm-toolchain-xenial-8
+      - sourceline: 'deb https://deb.nodesource.com/node_10.x bionic main'
+        key_url: 'https://deb.nodesource.com/gpgkey/nodesource.gpg.key'
+    packages:
+      - clang-8
+      - llvm-8-dev
+      - libclang-8-dev
+      - lld-8
+      - gcc-arm-linux-gnueabihf
+      - binutils-arm-none-eabi
```

Comment:
I believe binutils are not necessary. Have you tried running without?

Reply:
Trying without binutils.